### PR TITLE
Diagnostic Report Create for Cardiology and Anatomic pathology

### PR DESCRIPTION
**Pull request by Someshnsn**
Description
----

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
Screenshot 2023-11-02 at 4 04 43 PM
Screenshot 2023-11-02 at 4 04 52 PM
Screenshot 2023-11-02 at 4 04 59 PM
Screenshot 2023-11-02 at 4 05 09 PM
Screenshot 2023-11-02 at 4 05 17 PM
Screenshot 2023-11-02 at 4 05 23 PM
Screenshot 2023-11-02 at 4 05 31 PM
Screenshot 2023-11-02 at 4 05 37 PM
Screenshot 2023-11-02 at 4 05 57 PM
Screenshot 2023-11-02 at 4 06 08 PM
Screenshot 2023-11-02 at 4 08 47 PM
Screenshot 2023-11-02 at 4 08 53 PM
Screenshot 2023-11-02 at 4 09 00 PM

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Also "Body Fields" are repeated. Keep only 1 such menu

File: content/millennium/r4/clinical/diagnostics/diagnostic-report.md [hunk 1]
```diff
@@ -164,3 +164,117 @@ The common [errors] and [OperationOutcomes] may be returned.
...
+
+## Create
+
+Create a new diagnostic report.
+
+    POST /DiagnosticReport
+
+_Implementation Notes_
+
+* Concept Cardiology and Anotomic Pathology are supported for the create operation.
```
[the remark this answers is not in the document]
Comment (Someshnsn) [reply content/millennium/r4/clinical/diagnostics/diagnostic-report.md]: Addressed

File: content/millennium/r4/clinical/diagnostics/diagnostic-report.md [hunk 2]
```diff
@@ -164,3 +164,117 @@ The common [errors] and [OperationOutcomes] may be returned.
...
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* All provided dates must have a time component.
+* Supported MIME Types: application/pdf, text/plain, text/richtext, text/rtf, text/html, application/xml, and application/xhtml+xml
+* Diagnostic Report cardiology Write API requires
```
[the remark this answers is not in the document]
Comment (Someshnsn) [reply content/millennium/r4/clinical/diagnostics/diagnostic-report.md]: Addressed

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Please add examples from C1941 which the users can retry if needed. Currently it's from S1941

**Comment by Someshnsn** [comment]
> Also "Body Fields" are repeated. Keep only 1 such menu

Addressed

**Comment by Someshnsn** [comment]
> Please add examples from C1941 which the users can retry if needed. Currently it's from S1941

Addressed

**Review by ashu-bdvt — APPROVED** [review]

**Review by vinuthnag — APPROVED** [review]

**Pull request merged**